core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1013,16 +1029,6 @@ func (o *evmTxStore) UpdateTxCallbackCompleted(ctx context.Context, pipelineTask
 	return nil
 }
 
-func (o *evmTxStore) FindLatestSequence(ctx context.Context, fromAddress common.Address, chainId *big.Int) (nonce evmtypes.Nonce, err error) {
```

Comment:
Let's keep using this method and UpdateKeyNextSequence until we refactor the TxStore.

Reply:
Just wanted to note `UpdateKeyNextSequence` is unused. I removed it just as a clean up. It's actually updating a table that we don't own.